app_api/models/route.py

```diff
@@ -0,0 +1,137 @@
+from sqlalchemy import (
+    Column,
+    Integer,
+    SmallInteger,
+    String,
+    ForeignKey,
+    Enum
+    )
+
+from colanderalchemy import SQLAlchemySchemaNode
+
+from . import schema
+from utils import copy_attributes
+from document import (
+    ArchiveDocument, Document, DocumentLocale, ArchiveDocumentLocale)
+
+# TODO: move to a common place for outings etc.
+activities = [
+    'skitouring',
+    'snow_ice_mixed',
+    'mountain_climbing',
+    'rock_climbing',
+    'ice_climbing',
+    'hiking',
+    'snowshoeing',
+    'paragliding',
+    'mountain_biking',
+    'via_ferrata'
+    ]
```

Comment:
Yes i'ts used at line 34.
"Activities" is one of the main attributes of routes and outings. See for instance: http://www.camptocamp.org/routes/54417/fr/petite-dent-de-veisivi-traversee-des-aretes

By the way routes/outings may have several activities at the same time (for instance: http://www.camptocamp.org/routes/46065/fr/cheval-blanc-depuis-emosson)
=> not sure the current Enum type is correct?